types/artifacts.d.ts

```diff
@@ -483,6 +483,7 @@ declare global {
         nodeLabel?: string;
         devtoolsNodePath: string;
         snippet?: string;
+        score?: number;
```

Comment:
Open to suggestions on renaming this. Don't know if there's a better naming option here so left it as score for now.

Reply:
yeah this seems fine to me. `score` will be specific to the type of element, so `metricName` becomes a discriminant. future metrics may overload `score` but it'll be orthogonal so it seems fine to just use `score` and not `cls_score`. also seems fine to keep the types simple and not do a real type discriminant (like `interface CLSTraceElement extends TraceElement { score: number}`.